src/easyscore.js

```diff
@@ -0,0 +1,264 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+//
+// This class implements a parser for a simple language to generate
+// VexFlow objects.
+
+import { Vex } from './vex';
+import { StaveNote } from './stavenote';
+import { Parser } from './parser';
+
+// To enable logging for this class. Set `Vex.Flow.EasyScore.DEBUG` to `true`.
+function L(...args) { if (EasyScore.DEBUG) Vex.L('Vex.Flow.EasyScore', args); }
+
+export class X extends Error {
+  constructor(message) {
+    super(message);
+    this.message = message;
+    this.name = 'EasyScore';
+  }
+}
+
+function unquote(str) { return str.slice(1, -1); }
+
+class Grammar {
+  constructor(builder) {
+    this.builder = builder;
+  }
+
+  begin() { return this.LINE; }
+
+  // Notation grammar for EasyScore.
+  LINE()   {
+    return { expect: [this.PIECE, this.PIECES, this.EOL] };
+  }
+  PIECE()  {
+    return { expect: [this.CHORDORNOTE, this.PARAMS],
```

Comment:
Done for all the `expect` rules. I left `token` rules vertically aligned because they're more concise and easier to read that way.